packages/venia-ui/lib/components/Wishlist/WishlistDialog/NewWishlistForm/newWishlistForm.js

```diff
@@ -0,0 +1,101 @@
+import React, { Fragment } from 'react';
+import { useIntl } from 'react-intl';
+
+import { useNewWishlistForm } from '@magento/peregrine/lib/talons/Wishlist/WishlistDialog/NewWishlistForm/useNewWishlistForm';
+
+import Button from '@magento/venia-ui/lib/components/Button';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+import Field from '@magento/venia-ui/lib/components/Field';
+import TextInput from '@magento/venia-ui/lib/components/TextInput';
+import RadioGroup from '@magento/venia-ui/lib/components/RadioGroup';
+import { isRequired } from '@magento/venia-ui/lib/util/formValidators';
+
+import defaultClasses from './newWishlistForm.css';
+import FormError from '../../../FormError';
+
+const NewWishlistForm = props => {
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const { formatMessage } = useIntl();
+    const talonProps = useNewWishlistForm({
+        onCancel: props.onCancel,
+        onCreateList: props.onCreateList,
+        isAddLoading: props.isAddLoading
+    });
+
+    const { formErrors, handleCancel, handleSave, isSaveDisabled } = talonProps;
+
+    const privateRadioText = formatMessage({
+        id: 'newWishlistForm.privateRadio',
+        defaultMessage: 'Private'
+    });
+
+    const publicRadioText = formatMessage({
+        id: 'newWishlistForm.publicRadio',
+        defaultMessage: 'Public'
+    });
+
+    const cancelButtonText = formatMessage({
+        id: 'newWishlistForm.createButton',
```

Comment:
Should this be `.cancelButton`?

Reply:
Yes! Oops.

Reply:
Funny enough, the id was wrong but the text was right, so visually worked. Good catch :D